torch/_dynamo/convert_frame.py

```diff
@@ -267,6 +267,7 @@ def exception_handler(e, code, frame=None):
 def convert_frame_assert(
     compiler_fn: CompilerFn,
     guard_export_fn=None,
+    guard_fail_fn=None,
```

Comment:
Let's introduce some sort of options/hooks data structure, so we don't need so much plumbing to pass these around.

Reply:
Done